testsuite/cluster-test/src/experiments/mod.rs

```diff
@@ -129,6 +131,7 @@ pub fn get_experiment(name: &str, args: &[String], cluster: &Cluster) -> Box<dyn
         "reboot_random_validators",
         f::<RebootRandomValidatorsParams>(),
     );
+    known_experiments.insert("twin_validator", f::<TwinValidatorsParams>());
```

Comment:
Let's just use `twin` as a short name - shorter the better(experiment struct name can stay the same)